plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/LineRegion.java

```diff
@@ -36,6 +36,6 @@ public ILineRegion getNextLine() {
 
 	@Override
 	public ILineRegion getPreviousLine() {
-		return getTextRegionAccess().lineForOffset(getOffset() -1 );
+		return getTextRegionAccess().lineForOffset(getOffset() -2 );
```

Comment:
fixed in d1b505bad03c95a61808d9a4a7409f2357d339ad


On 15 Apr 2015, at 16:14, Sebastian Zarnekow <notifications@github.com> wrote:

> In plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/LineRegion.java:
> 
> > @@ -36,6 +36,6 @@ public ILineRegion getNextLine() {
> >  
> >  	@Override
> >  	public ILineRegion getPreviousLine() {
> > -		return getTextRegionAccess().lineForOffset(getOffset() -1 );
> > +		return getTextRegionAccess().lineForOffset(getOffset() -2 );
> Hmm this looks really strange. Could you document why the prev line is actually line-2 ?
> 
> —
> Reply to this email directly or view it on GitHub.
>